props/saber_fett263_buttons.h

```diff
@@ -5353,35 +5341,30 @@ SaberFett263Buttons() : PropBase() {}
       case EVENTID(BUTTON_NONE, EVENT_TWIST, MODE_ON):
         if (!saved_gesture_control.twistoff) return true;
         if (menu_) return true;
-        // Delay twist events to prevent false trigger from over twisting
-        if (millis() - last_twist_millis_ > 3000) {
 #ifdef FETT263_SAVE_CHOREOGRAPHY
-          if (choreo_) {
-            if (saved_choreography.clash_rec[clash_count_].stance == SavedRehearsal::STANCE_END) {
-            if (SFX_clsh) {
-              SFX_clsh.Select(-1);
-            } else {
-              SFX_clash.Select(-1);
-            }
-            if (SFX_lock) {
-              SFX_lock.Select(-1);
-            } else {
-              SFX_lockup.Select(-1);
-            }
-            SFX_bgnlock.Select(-1);
-            SFX_endlock.Select(-1);
-            choreo_ = false;
-            } else {
-              return true;
-            }
+        if (choreo_) {
```

Comment:
There seem to be a lot of stuff in here that is not related to spam delays...

Reply:
yeah, I tried doing the "branch" to have 2 PRs submitted on same file but this is what github keeps doing, it's showing the other PR in here as well, but if I don't branch the other PR then when you merge that it won't let this one go in.  That's why I've been doing one at a time.

Reply:
I really don't know what you're actually doing, but you need to branch from something that has NO changes in it.
What his means is that you need to stop making your edits on the master branch of your fork. That way the master branch on your fork is clean (identical to mine). From that master branch, you can create multiple branches in your fork, each with their own changes, and each can be submitted as a PR against my repository.

Reply:
I did that before but then when you merged the other PR it tells me that the 2nd PR can't merge because of differences in the files.  I can try again.

Reply:
When that happens, you just have to pull my changes onto your clean master, then do a merge on to the other branche(s) and re-upload. It might not be completely automatic, but unless there are actual conflicts it should be quick and easy to do.

Reply:
ok, I'll try it out submitted separate PR and have clean master already in place

Reply:
Btw, it doesn't have to be "master" that's the clean branch. It could be any branch. (You could call it "clean" for instance.)
You just have to set up the clean branch to pull directly from my repository and not make any changes on it.
Then make your branches from there.